### ISSUE TITLE
3MF v1.1 is here!

### ISSUE THREAD

**Issue by ghost** [Ultimaker/Cura#1363]
See http://3mf.io/3mf-version-1-1-is-here/ for the details, but all in all nothing major has changed, so 1.0 and 1.1 can be used interchangably,

Changes are as follows:

* Page 13: Restriction to absolute paths within the package. This was already true in the XSD schema, but is now reminded in the text.
* Page 15: Restricted XML content to UTF-8 to simplify parsers. We added a reminder of how XML namespace inheritance works.
* Pages 25-28: Added object type “solidsupport,” which like a model, must be manifold (hence solid). Objects of type “support” are not intended as solid objects, but as thin walls (regardless of whether they are manifold or not) to be manufactured at the machine’s precision. This ensures no ambiguity in whether support structures should be filled or not.
* Page 31: Restriction that base materials cannot form gradients, in order to ensure that splitting edges in a core-spec 3MF still results in a core-spec 3MF, rather than requiring composite materials from the materials and properties extension.
* Page 46: The only change to the XSD schema is to add “solidsupport.”

**Comment by Ghostkeeper** [comment]
The only change that Cura could implement is that new "solidsupport" feature. But since Cura currently doesn't support similar features either such as the gradient infill stuff, I don't think there is much haste to implement that.

**Comment by rogersachan** [comment]
See http://3mf.io/wp-content/uploads/2016/03/3MFcoreSpec_1.1.pdf for the document, since the link in page is broken.

**Comment by superusercode** [comment]
@nallath Was the feature added?

**Comment by Ghostkeeper** [comment]
No, we don't have solidsupport.

**Comment by rogersachan** [comment]
Did this ever land?

**Comment by Ghostkeeper** [comment]
No, we still don't have solidsupport-support. We have the feature in Cura but it's not automatically enabled when you load a 3MF file in Cura.

The gradient infill is in a side track under development though (but that will take a while still).

**Comment by rogersachan** [comment]
Is there a tracking issue for solidsupport-support?

**Comment by Ghostkeeper** [comment]
Yes, this one.

**Comment by GregValiant** [comment]
I'll go ahead and close this.  If you feel strongly about it then it can be reopened.
